### PR TITLE
Remove the CLI Commands Section from the Learn Page

### LINKED ISSUE #56: Remove the CLI Commands Section from the Learn Page
**Description:**
We need to improve the structure of the Learn page.

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
We need to revamp the Learn page to improve it.
> Fixes #56 and #83.

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/cli-commands.md [hunk 1]
```diff
@@ -6,82 +6,119 @@ permalink: /learn/cli-commands/
...
+These commands help you check your installation and know that things are set up smoothly.
+
+<table class="cComandTable">
+<tr>
+<td class="cCommand">`version`</td>
+<td class="cDescription">Tells you the version of Ballerina you have installed.</td>
+</tr>
+<tr>
+<td class="cCommand">home</td>
+<td class="cDescription">Prints out the location of you current Ballerina distribution. This is useful if you need to help your IDE plugin to figure out where Ballerina is.
```
Comment (anupama-pathirage) [inline learn/cli-commands.md]:
```suggestion
<td class="cDescription">Prints out the location of your current Ballerina distribution. This is useful if you need to help your IDE plugin to figure out where Ballerina is.
```

File: learn/cli-commands.md [hunk 2]
```diff
@@ -6,82 +6,119 @@ permalink: /learn/cli-commands/
...
+- [Work with Ballerina Central](#work-with-ballerina-central)
+- [Manage projects](#manage-projects)
+- [Use the supporting tools](#use-the-supporting-tools)
+
+## Get started
+These commands help you check your installation and know that things are set up smoothly.
+
+<table class="cComandTable">
+<tr>
+<td class="cCommand">`version`</td>
```
Comment (anupama-pathirage) [inline learn/cli-commands.md]:
```suggestion
<td class="cCommand">version</td>
```

File: learn/cli-commands.md [hunk 3]
```diff
@@ -6,82 +6,119 @@ permalink: /learn/cli-commands/
...
+<tr>
+<td class="cCommand">push</td>
+<td class="cDescription">Upload a module to Ballerina Central.
+</td>
+</tr>
+</table>
+
+## Manage projects
+
+Ballerina projects are the way to organize real world Ballerina development tasks. Learn more about it in <a href="/learn/how-to-structure-ballerina-code">How to Structure Ballerina Code</a>.
```
Comment (anupama-pathirage) [inline learn/cli-commands.md]:
```suggestion
Ballerina projects are the way to organize real-world Ballerina development tasks. Learn more about it in <a href="/learn/how-to-structure-ballerina-code">How to Structure Ballerina Code</a>.
```

File: learn/cli-commands.md [hunk 4]
```diff
@@ -6,82 +6,119 @@ permalink: /learn/cli-commands/
...
+<tr>
+<td class="cCommand">new</td>
+<td class="cDescription">Create a Ballerina project.
+</td>
+</tr>
+</table>
+
+## Use the supporting tools
+
+These powerful supporting tools extend Ballerina to various ecosystem technologies that are inherently cloud native. This functionality will grow over time and will even be developer extensible in the future.
```
Comment (anupama-pathirage) [inline learn/cli-commands.md]:
```suggestion
These powerful supporting tools extend Ballerina to various ecosystem technologies that are inherently cloud-native. This functionality will grow over time and will even be developer extensible in the future.
```

File: learn/cli-commands.md (hunk 3 above)
Comment (praneesha) [reply learn/cli-commands.md]: The hyphen is not needed when saying "real world".

Screenshot 2019-12-02 at 20 41 50

**Pull request merged**